client/my-sites/patterns/components/get-started/style.scss

```
@@ -33,8 +33,12 @@
 	flex: 1;
 
 	.patterns-get-started__item-image {
+		aspect-ratio: 16 / 9;
+		border-radius: 4px;
+		margin-bottom: 16px;
+		object-fit: cover;
+		overflow: hidden;
```

Don't see necessity, it seems they are redundant
```suggestion
		border-radius: 4px;
		margin-bottom: 16px;
```